docs/source/release_notes.rst

```diff
@@ -27,9 +29,12 @@ v0.5.0 Jul 7, 2021
         * Add ``infer_datetime_format`` param to speed up ``to_datetime`` calls (:pr:`1016`)
         * The default logical type is now the ``Unknown`` type instead of the ``NaturalLanguage`` type (:pr:`992`)
         * Add pandas 1.3.0 compatibility (:pr:`987`)
+    * Documentation Changes
+    * Testing Changes
 
     Thanks to the following people for contributing to this release:
-    :user:`jeff-hernandez`, :user:`simha104`, :user:`tamargrey`, :user:`thehomebrewnerd`, :user:`tuethan1999`
+    :user:`jeff-hernandez`, :user:`simha104`, :user:`tamargrey`,
+    :user:`thehomebrewnerd`, :user:`tuethan1999`
```

Comment:
I think we should revert these changes, so we are only updating the Future Release section.

Reply:
Ahh, sorry didn't catch that.  Thanks!